packages/core/src/components/axes/grid-brush.ts

```diff
@@ -49,13 +82,25 @@ export class ChartBrush extends Component {
 			this.frontSelectionSelector
 		);
 
+		const setDomain = (newDomain) => {
+			this.model.set({ zoomDomain: newDomain }, { animate: false });
+		};
+
+		const getDefaultZoomBarDomain = () =>
```

Comment:
is this really necessary?

Reply:
It helps to shorten function call a little bit, but I am ok if you don't think it makes sense.
Updated in https://github.com/carbon-design-system/carbon-charts/pull/717/commits/53e62c6aaa6c991e2a2197ea77f0920e1aa845d8